user-service/src/main/java/uk/gov/gchq/palisade/service/user/config/StdUserConfiguration.java

```diff
@@ -49,19 +49,19 @@ public StdUserConfiguration() {
      * @param users a list of objects implementing the {@link UserPrepopulationFactory} interface
      */
     public StdUserConfiguration(final List<StdUserPrepopulationFactory> users) {
-        this.users = users;
+        this.users = Collections.unmodifiableList(users);
```

Comment:
Have a setter that checks to make sure that the collection is not null.  For continuity, this would need to be the same.

Reply:
This was changed due to the sq smell of not using a setter method within a constructor. There might be a better way of doing this though.

Reply:
This is now using `List.copyOf()` to store a copy of the list